### ISSUE TITLE
DOC: Conda installation

### ISSUE THREAD

**Issue by dhomeier** [glue-viz/glue#2473]
**Describe the bug**
Is the installation procedure described on https://glueviz.readthedocs.io/en/stable/installation/conda.html still in any way relevant, in particular the `install -c glueviz glueviz` part? That channel never seems to have been updated beyond glue-core 1.2 and is probably broken entirely now, see also glue-viz/glue-standalone-apps#13 .
For comparison `conda install -c conda-forge --update-all glue-core` currently provides 1.17.1 among others (from miniforge/mambaforge, not sure how well this works out of a full anaconda installation).

**Comment by jfoster17** [comment]
I'm 99% sure that we should purge all references to the glueviz channel from our documentation and just recommend that people get glue from the conda-forge channel.

**Comment by astrofrog** [comment]
Yes this is correct, we should no longer use the glueviz channel
